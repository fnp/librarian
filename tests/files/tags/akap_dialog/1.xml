<akap_dialog>--- Nel, potrafisz wyliczyć nasze podróże od Fajumu? --- pytał Staś.</akap_dialog>

<akap_dialog>--- Potrafię.</akap_dialog>

<akap>To mówiąc dziewczynka podniosła w górę brwi i zaczęła rachować na paluszkach.</akap>

<akap_dialog>--- Zaraz. Od Fajumu do Chartumu --- to jedna; od Chartumu do Faszody --- to druga; od Faszody do tego wąwozu, w którym znaleźliśmy
Kinga --- to trzecia; a od Góry Lindego do jeziora --- to czwarta.</akap_dialog>

<akap_dialog>--- Tak. Chyba nie ma na świecie drugiej muchy, która by przeleciała taki kawał Afryki.</akap_dialog>

<akap_dialog>--- Ładnie by ta mucha wyglądała bez ciebie!</akap_dialog>
