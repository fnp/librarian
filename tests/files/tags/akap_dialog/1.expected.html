<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">— Nel, potrafisz wyliczyć nasze podróże od Fajumu? — pytał Staś.</p>
<a href="#f2" class="wl-num">2</a><p class="wl paragraph" id="f2">— Potrafię.</p>
<a href="#f3" class="wl-num">3</a><p class="wl paragraph" id="f3">To mówiąc dziewczynka podniosła w górę brwi i zaczęła rachować na paluszkach.</p>
<a href="#f4" class="wl-num">4</a><p class="wl paragraph" id="f4">— Zaraz. Od Fajumu do Chartumu — to jedna; od Chartumu do Faszody — to druga; od Faszody do tego wąwozu, w którym znaleźliśmy
Kinga — to trzecia; a od Góry Lindego do jeziora — to czwarta.</p>
<a href="#f5" class="wl-num">5</a><p class="wl paragraph" id="f5">— Tak. Chyba nie ma na świecie drugiej muchy, która by przeleciała taki kawał Afryki.</p>
<a href="#f6" class="wl-num">6</a><p class="wl paragraph" id="f6">— Ładnie by ta mucha wyglądała bez ciebie!</p>
</div>
